src/__tests__/utils.test.ts

```diff
@@ -160,6 +161,22 @@ describe('isStableRelease util', () => {
 	})
 })
 
+describe('sanitizeReleaseGroupTitle', () => {
```

Comment:
This query string shows results in production `comparator?repo=facebook%2Fdocusaurus&to=latest&from=v1.0.3` but fails in my Gitpod server preview:

## Unhandled Runtime Error
`TypeError: repository.owner is undefined`

### Source

`src/utils.ts (18:9) @ repository`

```bash
  16 | ): RepositoryQueryParams {
  17 | 	return {
> 18 | 		owner: repository?.owner.login ?? '',
     | 		      ^
  19 | 		repo: repository?.name ?? '',
  20 | 	}
  21 | }
```

### Call Stack

**params**
src/queries/release.tsx (29:2)

**ReleaseVersionsRangeFormControl**
src/components/ReleaseVersionsRangeFormControl.tsx (44:57)

**renderWithHooks**
node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom.development.js (16305:0)

**mountIndeterminateComponent**
node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom.development.js (20074:0)

**beginWork**
node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom.development.js (21587:0)

**callCallback**
node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom.development.js (4164:0)

**invokeGuardedCallbackDev**
node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom.development.js (4213:0)

**invokeGuardedCallback**
node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom.development.js (4277:0)

**beginWork$1**
node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom.development.js (27451:0)

**performUnitOfWork**
node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom.development.js (26557:0)

**workLoopSync**
node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom.development.js (26466:0)

**renderRootSync**
node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom.development.js (26434:0)

**performSyncWorkOnRoot**
node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom.development.js (26085:0)

**flushSyncCallbacks**
node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom.development.js (12042:0)

**ensureRootIsScheduled**/<
node_modules/.pnpm/react-dom@18.2.0_react@18.2.0/node_modules/react-dom/cjs/react-dom.development.js (25651:0)

Reply:
That's weird, it works on the preview env. Have you tried there?

Reply:
I think I just discovered why: by default, the requests are mocked in local envs, so the repo filled in the URL is not found. To unmock the GitHub requests, you need to disable the corresponding env var from `.env` in your `.env.local` and restart the local server.